.idea/vcs.xml

```diff
@@ -1,36 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where is this coming from ?

Reply:
Somehow my idea file got added and pushed. I think I removed it. Isn't it? Let me check again.